packages/dm-core-plugins/src/generic-list/GenericTablePlugin.tsx

```diff
@@ -0,0 +1,190 @@
+          .then(() =>
+            setItems({
+              ...items,
+              [crypto.randomUUID()]: { ...newEntity.data },
+            })
+          )
+      })
+  }
+
+  const updateItem = (
+    key: string,
+    attribute: string,
+    newValue: string | number | boolean
+  ) => {
+    items[key][attribute] = newValue
+    setItems({ ...items })
+  }
+
+  const saveTable = () => {
+    setIsSaveLoading(true)
+    dmssAPI
+      .documentUpdate({
+        idReference: idReference,
+        data: JSON.stringify(Object.values(items)),
+      })
+      .catch((error: AxiosError<ErrorResponse>) =>
+        alert(JSON.stringify(error.response?.data))
+      )
+      .finally(() => setIsSaveLoading(false))
+  }
+
+  if (loading) return <Loading />
+  if (error) throw new Error(JSON.stringify(error, null, 2))
+  if (!items) return <Loading />
+
+  return (
+    <>
+      <div
+        style={{ display: 'flex', flexDirection: 'row-reverse', width: '100%' }}
+      >
+        <Button
+          variant="ghost_icon"
+          onClick={() => addItem(idReference, 'type')}
+        >
+          <Icon data={library_add} title="Append" />
+        </Button>
+      </div>
+      <Table style={{ width: '100%' }}>
+        <Table.Head>
+          <Table.Row>
+            {internalConfig.columns.map((attribute: string) => (
+              <Table.Cell key={attribute}>{attribute}</Table.Cell>
+            ))}
+            {internalConfig.editMode && internalConfig.showDelete && (
+              <Table.Cell>Delete</Table.Cell>
+            )}
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>
+          {Object.entries(items).map(([key, item], index) => (
+            <Table.Row key={key}>
+              {internalConfig.columns.map((attribute: string) => {
+                if (typeof item[attribute] === 'object')
+                  throw new Error(
+                    `Objects can not be displayed in table header. Attribute '${attribute}' is not a primitive type.`
+                  )
+                // TODO: Consider having a more robust way of getting type
+                const attributeType = typeof item[attribute]
+                return (
+                  <Table.Cell key={attribute}>
+                    <Input
+                      value={item[attribute] || ''}
+                      readOnly={
+                        attribute === 'type' || !internalConfig.editMode
+                      }
+                      type={attributeType}
+                      rightAdornments={
+                        <>{internalConfig.editMode ? attributeType : ''}</>
+                      }
+                      onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                        let newValue: string | number | boolean =
+                          event.target.value
+                        if (attributeType === 'number')
+                          newValue = Number(newValue)
```

Comment:
I think this will cause some issues if you try to write 0.1. That's because Number("0.") == 0, so you won't be able to write the dot

Reply:
Writing "0.1" work fine. Both in chrome and firefox

Reply:
Sorry. Latest changes needed for that